app/views/rules_of_origin/_tab.html.erb

```diff
@@ -15,14 +15,15 @@
       <%= render 'rules_of_origin/rules_table',
                  country_name: country_name,
                  rules: rules %>
+
+      <%= render partial: 'rules_of_origin/introductory_notes',
+                 collection: rules_of_origin.map(&:introductory_notes) %>
```

Comment:
maybe adding `introductory_notes` as a public method to get them, instead of map here? 🤔

Reply:
Problem is rules of origin is an `Array` of `RulesOfOrigin::Scheme` not a specific class, so not sure where I would add it?